packages/pds/src/api/com/atproto/admin/sendEmail.ts

```diff
@@ -29,6 +31,20 @@ export default function (server: Server, ctx: AppContext) {
         { content },
         { subject, to: userInfo.email },
       )
+      await ctx.appViewAgent.api.com.atproto.admin.emitModerationEvent(
+        {
+          event: {
+            $type: 'com.atproto.admin.defs#modEventEmail',
+            subjectLine: subject,
+          },
+          subject: {
+            $type: 'com.atproto.admin.defs#repoRef',
+            did: recipientDid,
+          },
+          createdBy: senderDid,
+        },
+        { ...authPassthru(req), encoding: 'application/json' },
+      )
```

Comment:
I think we might want to make the email flow more uniform with how takedowns, etc. work: a `#modEventEmail` is dispatched to the appview using the `emitModerationEvent` endpoint, then the appview reaches out to `com.atproto.admin.sendEmail` on the PDS in order to push the email out to the user.